src/app/my-donations/my-donations.component.scss

```diff
@@ -0,0 +1,37 @@
+@import '../../abstract';
+@import "../../assets/scss/auth-common";
+
+#banner {
+  background-image: url(../../assets/images/banner-my-account.jpg);
+}
+
+.heading-wrapper {
+  margin: auto;
+  width: fit-content;
+}
+
+h1 {
+  color: $colour-secondary;
+}
+
+a:link, a:visited, a:hover {
+  color: white;
+}
+
+main > div {
+  color: white;
+  background-color: $colour-primary;
+  width: 100%;
+
+  @media #{$breakpoint-lg} {
+    width: 80%; // this seems to be needed, not sure why the identical rule in auth_common.scss isn't working.
+  }
+
+}
+
+// code above here mostly copied from my-account.component.scss
```

Comment:
Maybe a good time to add a mixin instead?

Reply:
I think that's pretty much what I was trying to do with auth-common: 

https://github.com/thebiggive/donate-frontend/blob/a6ed8d9c90ba94a1b1b658271a08eccf89658b03/src/assets/scss/auth-common.scss#L1 

but not sure why I still ended up needing to duplicate some stuff. I've done what I can for now to reduce duplication in 3d6016b4